### PR TITLE
Remove some Travis-CI references

### PR DESCRIPTION
**Pull request by RDIL**
# PR Summary
outta here travis-ci

## PR Context

I must complete the mission I started on January 14th, 2019

## PR Checklist

- [x] [PR has a meaningful title](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
    - Use the present tense and imperative mood when describing your changes
- [x] [Summarized changes](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
- [x] [Make sure all `.h`, `.cpp`, `.cs`, `.ps1` and `.psm1` files have the correct copyright header](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
- [x] This PR is ready to merge and is not [Work in Progress](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---work-in-progress).
    - If the PR is work in progress, please add the prefix `WIP:` or `[ WIP ]` to the beginning of the title (the `WIP` bot will keep its status check at `Pending` while the prefix is present) and remove the prefix when the PR is ready.
- **[Breaking changes](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#making-breaking-changes)**
    - [x] None
    - **OR**
    - [ ] [Experimental feature(s) needed](https://github.com/MicrosoftDocs/PowerShell-Docs/blob/staging/reference/6/Microsoft.PowerShell.Core/About/about_Experimental_Features.md)
        - [ ] Experimental feature name(s): 
- **User-facing changes**
    - [x] Not Applicable
    - **OR**
    - [ ] [Documentation needed](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
        - [ ] Issue filed: 
- **Testing - New and feature**
    - [x] N/A or can only be tested interactively
    - **OR**
    - [x] [Make sure you've added a new test if existing tests do not effectively test the code changed](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#before-submitting)

### CODE REVIEW

**Review by TravisEz13 — APPROVED** [review]

**Comment by TravisEz13** [comment]
@PoshChan Please remind me in 24 hours

**Comment by RDIL** [comment]
(Not to be confused with @TravisEz13 😆)

File: docs/testing-guidelines/TestRoadmap.md [hunk 1]
```diff
@@ -79,7 +79,6 @@ We need to be sure that we can easily enable remoting for the non-Windows platfo
 * Our current multi-machine tests do not test the connection code, they simply execute test code remotely and retrieve results and assume a good connection.
 The infrastructure used for these tests is STEX which is not an open environment.
 We will need to create automation to create and configure the test systems in the test matrix and then invoke tests on them.
-It is not clear that our current CI systems can accommodate our needs here as neither AppVeyor or Travis can supply us with all of the OS images needed.
```
Comment (TravisEz13) [inline docs/testing-guidelines/TestRoadmap.md]: This is still true of Azure DevOps as we have not solved windows 7 and some other Operating Systems.

**Review by TravisEz13 — CHANGES_REQUESTED** [review]
see comment

File: docs/testing-guidelines/TestRoadmap.md (hunk 1 above)
Comment (RDIL) [reply docs/testing-guidelines/TestRoadmap.md]: Fixed

File: docs/testing-guidelines/TestRoadmap.md [hunk 2]
```diff
@@ -79,7 +79,7 @@ We need to be sure that we can easily enable remoting for the non-Windows platfo
...
+It is not clear that our current CI systems can accommodate our needs here as Azure DevOps can't supply us with all of the OS images needed.
```
Comment (TravisEz13) [inline docs/testing-guidelines/TestRoadmap.md]:
```suggestion
It is not clear that our current CI systems can accommodate our needs here as Azure DevOps can supply us with all of the OS images needed.
```
remove double negative

**Comment by PoshChan** [comment]
@TravisEz13, this is the reminder you requested 24 hours ago

**Comment by TravisEz13** [comment]
@RDIL 	Why did you close and reopen the issue?

**Review by TravisEz13 — APPROVED** [review]

**Comment by RDIL** [comment]
Wrong button pressed

**Comment by RDIL** [comment]
Wrong button pressed

**Pull request merged**